Library/Homebrew/requirements/maximum_macos_requirement.rb

```diff
@@ -12,7 +12,11 @@ def initialize(tags)
 
   def message
     <<-EOS.undent
-      OS X #{@version.pretty_name} or older is required.
+      This formula either does not compile or function as expected on newer
```

Comment:
I think `or` is typically used in an `either`, and `nor` in `neither`?

* Dom found neither Mike nor Misty.
* Dom found either Mike or Misty.

Rather than

* Dom found either Mike nor Misty.
* Dom found neither Mike or Misty.

Maybe.